src/domain/ledger/index.types.d.ts

```diff
@@ -29,7 +29,7 @@ type LedgerTransaction = {
   readonly feeUsd: number
 
   // for IntraLedger
-  readonly username?: Username
+  readonly walletName?: WalletName
```

Comment:
are you sure if you do this will not affect current data? I guess you will need to add a migration

Reply:
I was careful to isolation these `LedgerTransaction` changes to one commit (https://github.com/GaloyMoney/galoy/pull/475/commits/f5e7786575da2d77690b704273325b66a674ab36) to double-check that we don't need migrations.

So far, this type is only used in two places:
- input of `fromLedger` where it gets translated to `ConfirmedTransactionHistory` type so `username`-naming is irrelevant
- return from `getLiabilityTransactions`, which only gets used as the input to the `fromLedger` method